tests/cupy_tests/lib_tests/test_strided_tricks.py

```diff
@@ -6,29 +6,98 @@
 from cupy import testing
 from cupy.lib import stride_tricks
 
+import pytest
+
 
 class TestAsStrided(unittest.TestCase):
     def test_as_strided(self):
         a = cupy.array([1, 2, 3, 4])
-        a_view = stride_tricks.as_strided(
-            a, shape=(2,), strides=(2 * a.itemsize,))
+        a_view = stride_tricks.as_strided(a, shape=(2,), strides=(2 * a.itemsize,))
         expected = cupy.array([1, 3])
         testing.assert_array_equal(a_view, expected)
 
         a = cupy.array([1, 2, 3, 4])
-        a_view = stride_tricks.as_strided(
-            a, shape=(3, 4), strides=(0, 1 * a.itemsize))
+        a_view = stride_tricks.as_strided(a, shape=(3, 4), strides=(0, 1 * a.itemsize))
         expected = cupy.array([[1, 2, 3, 4], [1, 2, 3, 4], [1, 2, 3, 4]])
         testing.assert_array_equal(a_view, expected)
 
     @testing.numpy_cupy_array_equal()
     def test_rolling_window(self, xp):
         a = testing.shaped_arange((3, 4), xp)
         a_rolling = rolling_window(a, 2, 0)
-
         return a_rolling
 
 
+class TestSlidingWindowView(unittest.TestCase):
+    @testing.numpy_cupy_array_equal()
+    def test_1d(self, xp):
+        arr = testing.shaped_arange((3, 4), xp)
+        window_size = 2
+        arr_view = xp.lib.stride_tricks.sliding_window_view(arr, window_size, 0)
+        return arr_view
+
+    @testing.numpy_cupy_array_equal()
+    def test_2d(self, xp):
+        arr = testing.shaped_arange((3, 4), xp)
+        window_shape = (2, 2)
+        arr_view = xp.lib.stride_tricks.sliding_window_view(
+            arr, window_shape=window_shape
+        )
+        return arr_view
+
+    @testing.numpy_cupy_array_equal()
+    def test_2d_with_axis(self, xp):
+        arr = testing.shaped_arange((3, 4), xp)
+        window_shape = 3
+        axis = 0
+        arr_view = xp.lib.stride_tricks.sliding_window_view(arr, window_shape, axis)
+        return arr_view
+
+    @testing.numpy_cupy_array_equal()
+    def test_2d_multi_axis(self, xp):
+        arr = testing.shaped_arange((3, 4), xp)
+        window_shape = (2, 3)
+        axis = (0, 1)
+        arr_view = xp.lib.stride_tricks.sliding_window_view(arr, window_shape, axis)
+        return arr_view
+
+    def test_0d(self):
+        # Create a 0-D array (scalar) for testing
+        arr = cupy.array(42)
+        # Sliding window with window size 1
+        window_size = 1
+
+        # Test if the correct ValueError is raised!
+        with pytest.raises(ValueError, match="axis 0 is out of bounds"):
+            cupy.lib.stride_tricks.sliding_window_view(arr, window_size, 0)
```

Comment:
Could you check the raised error messages is equal to NumPy's one?
```suggestion
        for xp in (numpy, cupy):
            # Create a 0-D array (scalar) for testing
            arr = xp.array(42)
            # Sliding window with window size 1
            window_size = 1

            # Test if the correct ValueError is raised!
            with pytest.raises(ValueError, match="axis 0 is out of bounds"):
                xp.lib.stride_tricks.sliding_window_view(arr, window_size, 0)
```